### PR TITLE
Various fixes

### PR DESCRIPTION
**Pull request by RaqManzano**
Various fixes and leftover pushes from previous branch.

### CODE REVIEW

**Comment by RaqManzano** [comment]
I have added here the Dockerfiles we talked about @maxulysse. Could you please take a look at let me know if I have missed anything?

File: modules/local/consensus/main.nf [hunk 1]
```diff
@@ -3,7 +3,7 @@ process RUN_CONSENSUS {
     label 'process_low'
 
     conda "bioconda::bioconductor-rtracklayer bioconda::bioconductor-complexheatmap conda-forge::r-ggrepel conda-forge::r-data.table conda-forge::r-dplyr conda-forge::ggpubr "
-    container 'ghcr.io/raqmanzano/renv:latest'
+    container 'ghcr.io/raqmanzano/renv:1.0'
```
Comment (maxulysse) [inline modules/local/consensus/main.nf]:
```suggestion
    container 'nf-core/rnadnavar_renv_consensus:1.0'
```

File: modules/local/maf_rna_filtering/main.nf [hunk 2]
```diff
@@ -3,7 +3,7 @@ process RNA_FILTERING {
     label 'process_low'
 
     conda null
-    container 'ghcr.io/raqmanzano/rnafilt:latest'
+    container 'ghcr.io/raqmanzano/rnafilt:1.0'
```
Comment (maxulysse) [inline modules/local/maf_rna_filtering/main.nf]:
```suggestion
    container 'nf-core/rnadnavar_pyenv_rnafilt:1.0'
```

File: modules/local/consensus/main.nf [hunk 3]
```diff
@@ -3,7 +3,7 @@ process RUN_CONSENSUS {
     label 'process_low'
 
     conda "bioconda::bioconductor-rtracklayer bioconda::bioconductor-complexheatmap conda-forge::r-ggrepel conda-forge::r-data.table conda-forge::r-dplyr conda-forge::ggpubr "
-    container 'ghcr.io/raqmanzano/renv:latest'
+    container 'ghcr.io/raqmanzano/rnadnavar_renv_consensus:1.0'
```
Comment (maxulysse) [inline modules/local/consensus/main.nf]:
```suggestion
    container 'nf-core/rnadnavar_renv_consensus:1.0'
```

File: modules/local/maf_rna_filtering/main.nf [hunk 4]
```diff
@@ -3,7 +3,7 @@ process RNA_FILTERING {
     label 'process_low'
 
     conda null
-    container 'ghcr.io/raqmanzano/rnafilt:latest'
+    container 'ghcr.io/raqmanzano/rnadnavar_pyenv_rnafilt:1.0'
```
Comment (maxulysse) [inline modules/local/maf_rna_filtering/main.nf]:
```suggestion
    container 'nf-core/rnadnavar_pyenv_rnafilt:1.0'
```

**Review by maxulysse — APPROVED** [review]
LGTM

**Pull request merged**